lib/stencil-bundle.js

```diff
@@ -278,6 +278,16 @@ function validateTheme(callback) {
     });
 }
 
+/**
+ * check for dist folder
+ * @param {*}
+ * @returns {Boolean}
+ * 
+ */
+Bundle.prototype.distFolderCheck = function() {
```

Comment:
Is it enough to only check for the folder existence? Do we remove this folder on every run?

Reply:
The issue was that the dist folder was not being generated so I thought this should be enough.  The folder is not removed on every run.

Reply:
I can put back the check for files but a for loop check would be difficult as bundle chunk files may go from 2 to 4 or not start off at 0(?) and also a check for the main file.

Reply:
If the dist folder is not removed on each run, then this check will always return true after the first run, even if the files are not generated properly.